worker/src/RTC/SimulcastConsumer.cpp

```diff
@@ -891,6 +891,8 @@ namespace RTC
 		// Process the packet.
 		if (this->rtpStream->ReceivePacket(packet))
 		{
+			this->lastSentPacketHasMarker = packet->HasMarker();
```

Comment:
I think this is not ok since RTP packets here could have arrived disordered to mediasoup. We should just set this flag (to true or false) if this is the highest seq number we have sent to the endpoint.

Reply:
correct

Reply:
You are right, i'll fix it